pom.xml

```diff
@@ -132,7 +132,11 @@
             <compilerArgs>
               <!-- Args related to Error Prone, see: https://errorprone.info/docs/installation#maven -->
               <arg>-XDcompilePolicy=simple</arg>
-              <arg>-Xplugin:ErrorProne -XepExcludedPaths:.*/generated-test-sources/protobuf/.*</arg>
+              <arg>-Xplugin:ErrorProne
+                -XepExcludedPaths:.*/generated-test-sources/protobuf/.*
+                -Xep:HidingField:OFF <!-- Work around NoSuchMethodError bug. -->
```

Comment:
It could be nice to cite https://github.com/google/error-prone/issues/3895 here or in the PR description, but it probably doesn't matter much.

Reply:
Thanks, done.